testutil/testutil.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// SetBucket sets the bucket in the environment to a local pail bucket
+// backed by a temporary directory.
+// If initDir is not the empty string the contents of initDir are copied into the local bucket.
```

Comment:
`If initDir is not empty the contents of the directory with the given path are copied to the local bucket.`